src/server/js/server.js

```diff
@@ -50,9 +49,12 @@ dirs(`${__dirname}/resources`).forEach(dir => {
     app.use(`/api/${dir}`, api);
 });
 
+app.use(bodyParser.json());
+app.use(passport.initialize());
+app.use('/', routes);
```

Comment:
is this how you are supposed to import routes?

Reply:
i don't think we should have these routes